engines/wintermute/base/font/base_font_bitmap.cpp

```diff
@@ -346,6 +361,10 @@ bool BaseFontBitmap::loadBuffer(char *buffer) {
 	int i;
 	int r = 255, g = 255, b = 255;
 	bool custoTrans = false;
+#ifdef ENABLE_FOXTAIL
+	int ar = 255, ag = 255, ab = 255;
+	bool custoAlpha = false;
```

Comment:
customAlpha

Reply:
What about `custoTrans`? This names is used in 3 files: base/base_frame.cpp, base/base_sub_frame.cpp, base/font/base_font_bitmap.cpp. 

Should I change both names in all files? Changing `custoAlpha` feels inconsistent.

Reply:
Renamed `custoAlpha` and `custoTrans` at 6b4e06b13ca2d697f8de80f3fab9b493590256e3.